packages/create/src/cli.ts

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env node
+
+import { Command } from 'commander';
+import inquirer from 'inquirer';
+
+const program = new Command();
+
+program
+  .name('create-sku')
+  .description('Create a new sku project')
+  .version('0.0.0')
+  .argument('[project-name]', 'Name of the project to create')
+  .option('-t, --template <template>', 'Template to use (webpack, vite)')
+  .action(async (projectName: string = '.', options: { template?: string }) => {
+    // Prompt for template if not provided via CLI option
+    const template =
+      options.template ||
+      (
+        await inquirer.prompt([
+          {
+            type: 'list',
+            name: 'template',
+            message: 'Which template would you like to use?',
+            choices: ['webpack', 'vite'],
+            default: 'webpack',
+          },
+        ])
+      ).template;
```

Comment:
This is a bit hard to read. IMO it would read better with an `if` statement:

```suggestion
    let selectedTemplate = options.template
    if (!template) {
      const { template } = await inquirer.prompt([
        {
          type: 'list',
          name: 'template',
          message: 'Which template would you like to use?',
          choices: ['webpack', 'vite'],
          default: 'webpack',
        },
      ]);
      selectedTemplate = template;
    };
```